site/source/docs/porting/simd.rst

```diff
@@ -1123,47 +1126,47 @@ For detailed information on each intrinsic function, refer to `NEON Intrinsics R
    * - vabdl
      - ⚫ Not implemented, will trigger compiler error
    * - vabs
-     - native
+     - ✅ native
    * - vadd
-     - native
+     - ✅ native
    * - vaddl
      - ⚫ Not implemented, will trigger compiler error
    * - vaddlv
      - ⚫ Not implemented, will trigger compiler error
    * - vaddv
      - ⚫ Not implemented, will trigger compiler error
-   * - vaddw 
+   * - vaddw
      - ❌ Will be emulated with slow instructions, or scalarized
-   * - vand 
-     - native
+   * - vand
+     - ✅ native
    * - vbic
      - ⚫ Not implemented, will trigger compiler error
    * - vbsl
-     - native
+     - ✅ native
    * - vcagt
-     - ⚠ Does not have direct implementation, but is emulated using fast NEON instructions
+     - ⚠️  Does not have direct implementation, but is emulated using fast NEON instructions
```

Comment:
Why have two spaces after this emoji but only one space after the check mark and X emojis?